Library/Homebrew/vendor/README.md

```diff
@@ -33,7 +33,7 @@ Vendored Dependencies
 ### ruby-macho
 
 > The MIT License
-> Copyright (c) 2015, 2016 William Woodruff <william @ tuffbizz.com>
+> Copyright (c) 2015, 2016, 2017 William Woodruff <william @ tuffbizz.com>
```

Comment:
General recommendation that the year be removed from the upstream software here. My software licensing legal friends tell me it's unnecessary and it'll help you avoid this busywork. Fine with it being changed here for now, though.